### ISSUE TITLE
compiler allows ";" before closing mxml tag

### ISSUE THREAD

**Issue by carlosrovira** [apache/royale-compiler#120]
This code does not cause error, notice the `;` before the closing tag:

```xml
<j:Button text="change something" click="someVar = 'other text'";/>
```

**Comment by joshtynjala** [comment]
Confirmed. I found a lot of special characters that seem to be allowed by the Royale compiler, but the Flex SDK compiler does not.

```xml
<j:Button text="change something" click="someVar = 'other text'" -,.{}?;/\()[]&@#!/>
```
